src/components/Editor/Invitees/InviteesList.vue

```diff
@@ -255,9 +255,8 @@ export default {
 		alreadyInvitedEmails() {
 			const emails = this.invitees.map(attendee => removeMailtoPrefix(attendee.uri))
 
-			const principal = this.principalsStore.getCurrentUserPrincipal
-			if (principal) {
-				emails.push(principal.emailAddress)
+			if (this.calendarObjectInstance.organizer.uri) {
+				emails.push(removeMailtoPrefix(this.calendarObjectInstance.organizer.uri))
```

Comment:
```suggestion
			// A user should be able to invite themselves if they are not the organizer
			const principal = this.principalsStore.getCurrentUserPrincipal
			const organizerUri = this.calendarObjectInstance.organizer?.uri
			if (organizerUri && removeMailtoPrefix(organizerUri) !== principal.emailAddress) {
				emails.push(removeMailtoPrefix(organizerUri))
			} else if (principal) {
				emails.push(principal.emailAddress)
			}
```

Only add the organizer's URI if it isn't equal to the current principal's email address. I should only be able to invite myself if I'm not the organizer.

This was tested locally. See my other comment above.